eZ/Publish/Core/FieldType/Validator/StringLengthValidator.php

```diff
@@ -63,6 +64,30 @@ public function validateConstraints($constraints)
                         );
                     }
                     break;
+                case 'defaultValue':
+                    if ($value instanceof TextLineValue) {
+                        $value = $value->text;
+                    }
+                    $minStringLength = $constraints['minStringLength'];
+                    $maxStringLength = $constraints['maxStringLength'];
+                    if ($minStringLength !== false && is_integer($minStringLength) && (strlen($value) < $minStringLength)) {
+                        $validationErrors[] = new ValidationError(
+                            "Validator parameter '%parameter%' length can't be shorter than min string length value",
```

Comment:
Nitpick: "min" isn't really a word, I believe we use "minimum" elsewhere. Same with "max" below.